newIDE/app/src/Export/GenericExporters/OnlineWebExport/OnlineGamePropertiesDialog.js

```diff
@@ -51,6 +59,12 @@ export const OnlineGamePropertiesDialog = ({
   const [playWithMobile, setPlayableWithMobile] = React.useState<boolean>(
     project.isPlayableWithMobile()
   );
+  const [userSlug, setUserSlug] = React.useState<string>(
+    (slug && slug.username) || (profile && profile.username) || ''
```

Comment:
If a user doesn't have a username defined, this will be empty.
But then, does it mean they cannot publish their game because they can't create an slug with empty username?
They will have to go back to their profile to add a username before coming back here?

Reply:
It wouldn't block the publishing but it will look like a broken UI. I will hide the slugs fields in this case.

Reply:
![image](https://user-images.githubusercontent.com/2611977/162154584-1a452416-5b59-4bd5-93a9-6455187b1377.png)

Reply:
I like the tip + the disabled fields 👍